src/tools/run-make-support/src/lib.rs

```diff
@@ -45,6 +48,11 @@ impl RustcInvocationBuilder {
         self
     }
 
+    pub fn args(&mut self, args: &[&str]) -> &mut RustcInvocationBuilder {
+        self.cmd.args(args);
+        self
+    }
```

Comment:
Not sure if there's a good way to encourage people to be more careful of paths with platform-specific separators (I commit path crimes too often). This is probably fine, maybe another PR sometime can add an `arg_path()` method or similar.

Reply:
Oh I was adding this as a parallel to the `fn arg` method above, but are you referring to the argument of `&[&str]` which should in theory be something more like `&[impl AsRef<OsStr>]` or similar? Or you think it's too much of a footgun to pass `"a/b"` here instead of `"a\\b"` on Windows?

Reply:
> Oh I was adding this as a parallel to the `fn arg` method above, but are you referring to the argument of `&[&str]` which should in theory be something more like `&[impl AsRef<OsStr>]` or similar? Or you think it's too much of a footgun to pass `"a/b"` here instead of `"a\\b"` on Windows?

This is probably fine tbh. I'm just paranoid of path-related shenanigans after recently commiting a path crime for tidy 😆 We can always add a `arg_path` helper method or something and document to prefer that over `args` when you are working with paths (if possible).

Reply:
> Or you think it's too much of a footgun to pass `"a/b"` here instead of `"a\\b"` on Windows?

It's always funny, because *some* tools/executables properly normalize/convert a Unix-y input path so they work on Windows properly, but other tools just give up.